parsimonious/grammar.py

```diff
@@ -95,8 +87,9 @@ def _copy(self):
 
         """
         new = Grammar.__new__(Grammar)
+        super(Grammar, new).__init__(self.items())
```

Comment:
Let's merge it as is, and maybe I'll refactor someday when I'm able to spend some real time on it.